modules/op-interacting-with-pipelines-using-the-developer-perspective.adoc

```diff
@@ -11,36 +11,29 @@ The *Pipelines* view in the *Developer* perspective lists all the Pipelines in a
 +
 //image::op-pipeline-details.png[Pipeline details]
 +
 . Optionally, in the *Pipeline Details* page:
 
+* Click the *YAML* tab to edit the YAML file for the Pipeline.
 * Click the *Pipeline Runs* tab to see the completed, running, or failed runs for the Pipeline. You can use the Options menu {kebab} to stop a running Pipeline, to rerun a Pipeline using the same parameters and resources as that of the previous Pipeline execution, or to delete a PipelineRun.
 * Click the *Parameters* tab to see the parameters defined in the Pipeline. You can also add or edit additional parameters as required.
 * Click the *Resources* tab to see the resources defined in the Pipeline. You can also add or edit additional resources as required.
 
-////
-. Click *Actions* and select *Start* to start the Pipeline. The *Start Pipeline* dialog box displays the *Git Resources* and the *Image Resources* based on the Pipeline definition.
-.. In the *Parameters* section, enter a unique name in the `IMAGE_NAME` field.
-.. Use the drop-down lists to select or create the required resources to customize the Pipeline as required. For this example, retain the default values and click *Start* to start the PipelineRun. After the Pipeline starts, the Tasks are executed sequentially in the Pipeline.
-//Specifically for this Pipeline `IMAGE_NAME` may end up as 'APP_NAME' to fulfil what they are looking for. Ticket: https://issues.redhat.com/browse/SRVKP-631 Update docs based on the bug resolution
-
-. The *Pipeline Run Details* page displays the pipeline being executed. You can:
-
-* Hover over the tasks to see the time taken for the execution of each step.
-* Click on a task to see logs for each of the steps in the task.
-* Click the *Logs* tab to see the logs according to the execution sequence of the tasks and use the *Download* button to download the logs to a text file.
+. For Pipelines created using the *From Git* option, you can also use the *Topology* view to interact with Pipelines:
 +
-image::pipeline_run.png[Pipeline run]
+[NOTE]
+====
+To see Pipelines created using the *Pipeline Builder* in the *Topology* view, customize the Pipeline labels to link the Pipeline with the application workload.
+====
 +
-. On the left navigation panel, click *Topology* and click on the application to see the pipeline runs listed in the right panel.
-. In the *Pipeline Runs* section, click *Start Last Run* to start a new pipeline run with the same parameters and resources as the previous ones. This option is disabled if a pipeline run has not been initiated.
+.. On the left navigation panel, click *Topology*, and click on the application to see the Pipeline Runs listed in the side panel.
+.. In the *Pipeline Runs* section, click *Start Last Run* to start a new Pipeline Run with the same parameters and resources as the previous ones. This option is disabled if a Pipeline Run has not been initiated.
 +
```

Comment:
Wonder if it's worth making a `[NOTE]` here that they can click on the Pipeline link or maybe view the "Starting a Pipeline" section? (this is probably only needed if you follow my previous suggestion on flipping the sections).

Reply:
So as per the above suggestion I flipped the sections, so now we have Creating via Builder>Creating via 'from git' (just a link)>interacting and then Start.
Therefore I had to move the topology section from the Interacting section to the Start section, as without starting a pipeline you will not see it in the Topology view. Hence note is not required.